packages/react/src/components/UIShell/HeaderMenuItem.js

```diff
@@ -8,19 +8,36 @@
 import { settings } from 'carbon-components';
 import PropTypes from 'prop-types';
 import React from 'react';
+import cx from 'classnames';
 import Link, { LinkPropTypes } from './Link';
 
 const { prefix } = settings;
 
 const HeaderMenuItem = React.forwardRef(function HeaderMenuItem(
-  { className, children, role, ...rest },
+  {
+    className,
+    isCurrentPage,
+    'aria-current': ariaCurrent,
+    children,
+    role,
+    ...rest
+  },
   ref
 ) {
+  const linkClassName = cx({
+    [`${prefix}--header__menu-item`]: true,
+    // We set the current class only if `isCurrentPage` is passed in and we do
+    // not have an `aria-current="page"` set for the breadcrumb item
+    [`${prefix}--header__menu-item--current`]:
+      isCurrentPage && ariaCurrent !== 'page',
```

Comment:
Usage of ARIA attribute content for styling has caused breaking changes during our a11y fix effort. So wondering if just `isCurrentPage` is enough.

Reply:
I don't think we'll run into any problems since it's being implemented in both ways. It definitely is risky to style using ARIA roles since we might change those based on new info but `aria-current="page"` is more stable and not likely to change. I also think it's nice to support both use cases for the ways that people might try to implement current page styling.

Reply:
> `aria-current="page"` is more stable and not likely to change

Probably it may change (esp. in post-IE era) because of newer spec in this area is on the horizon https://wicg.github.io/aom/explainer.html

Reply:
It seems like it'd be good to go for now, given the class fallback. If there is a specific line or section from the spec that you think is relevant and should be included in the PR that'd be great to highlight 👍 `aria-current` with `page` seems like a fairly consistent attribute for communicating present navigation to screen readers.